src/test/java/com/scottlogic/TopicFinderTest.java

```diff
@@ -0,0 +1,54 @@
+package com.scottlogic;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public  class TopicFinderTest {
+
+
+    private static List<String> stopWords;
+
+    private UserPost createUserPost(String content) {
+        return new UserPost("Joe Bloggs",
+                OffsetDateTime.of(2020, 1, 3, 7, 12, 3, 0, ZoneOffset.UTC),
+                content, 2);
+    }
+
+    @BeforeClass
+    public static void loadStopWords() throws IOException {
+        File file = new File("C:/Dev/post-sorting-grad-project/src/main/resources/English_StopWords.txt");
+       stopWords = Files.readAllLines(Paths.get(String.valueOf(file)));
+    }
+
+    TopicFinder topicFind = new TopicFinder(stopWords);
```

Comment:
I think the assignment part of this line should be in the method above. Less importantly, it might be nicer to have the declaration part after you declare `stopWords`.